src/Stryker.Core/Stryker.Core/Mutants/MutantOrchestrator.cs

```diff
@@ -62,12 +64,150 @@ public IEnumerable<Mutant> GetLatestMutantBatch()
             return tempMutants;
         }
 
+
+        private bool CanBeMutated(SyntaxNode node)
```

Comment:
Or rewrite this to an extension method. It might be nice to create a folder of `SyntaxNodeExtensions` that contains `CanBeMutated` and `IsStringLiteral`. We might think of even more. This would put all SyntaxNode logic in one place and clean up this huge class a bit.

Reply:
I thought about it, but then I realised that this method belongs to MutantOrchestrator as it relates to its inability to handle mutants.
To put it in another way: those node can be mutated, but they cannot be controlled by an If or a conditional expression.